UsbAccess/UsbAccess.cpp

```diff
@@ -702,5 +706,52 @@ namespace Plugin {
 
         return result;
     }
+
+    uint32_t UsbAccess::getUSBSizeWrapper(const JsonObject &parameters, JsonObject &response)
+    {
+        LOGINFOMETHOD();
+
+        bool result = false;
+        char cmd[128] = {'\0'};
+        char totalsize[20] = {'\0'};
+        char usedsize[20] = {'\0'};
+        char freesize[20] = {'\0'};
+
+        std::list<string> paths;
+        getMounted(paths);
+        if (!paths.empty()) {
+            snprintf(cmd, 127, "df -h | grep %s | awk \'{print $2 \" \" $3 \" \" $4}\'", (*paths.begin()).c_str());
+            FILE* fp = popen(cmd, "r");
+            if (fp) {
+                fscanf(fp,"%s %s %s", totalsize, usedsize, freesize);
+                response["totalsize"] = string(totalsize);
+                response["usedsize"] = string(usedsize);
+                response["freesize"] = string(freesize);
+                pclose(fp);
+                result = true;
+            }
+        }
+        returnResponse(result);
+    }
+
+    uint32_t UsbAccess::unmountUSBWrapper(const JsonObject &parameters, JsonObject &response)
```

Comment:
Pls help elaborate on use case for unmounting through the API. The mount of USB drives is also handled outside of the plugin.

Reply:
Both mount & unmount is handled by the udev scripts in case of RDKV https://github.com/rdkcentral/rdkservices/blob/main/UsbAccess/CMakeLists.txt#L9.
And the script is part of sysint.